pegasus/sites.v3/code.org/views/ai_header_stylesheets_links.erb

```diff
@@ -0,0 +1,8 @@
+<%
+  show_tune_in_link = DCDO.get('show_2020_ethics_ai_panel_live_links', true)
```

Comment:
is this used anywhere?

Reply:
It's not; I left it in in case this was some kind of future-proofing that I didn't have context on.

Reply:
This was from Hour of Code 2020, where we used it to hide some "tune in live" links after a [live event had aired](https://github.com/code-dot-org/code-dot-org/pull/38047/files). Looks like the flag usage was deleted since then, so I think you could remove, but not really your problem I guess :)

Reply:
I'll delete!